.github/workflows/publish-orca-server.yml

```
@@ -0,0 +1,167 @@
+# Auth: NODE_AUTH_TOKEN from the NPM_TOKEN repo secret (must exist in this repo,
+# same as stablyai/stably's release-cli). Set release_type=rc for a prerelease
+# published under the `alpha` dist-tag; omit (or 'release') for a normal publish.
+
+on:
+  workflow_dispatch:
+    inputs:
+      release_type:
+        description: 'Release type (rc → published under the alpha tag)'
+        required: false
+        type: choice
+        options: ['', rc, release]
```

actionlint (1.7.12)

[error] string should not be empty

(syntax-check)